tbx/static_src/sass/components/_grid.scss

```diff
@@ -81,6 +81,14 @@
         }
     }
 
+    &__button {
```

Comment:
Is this needed? I think it's fine to just have in the 404 page section

Reply:
I can see how it makes sense to just remove it. Removed in [6ad6a46](https://github.com/torchbox/torchbox.com/pull/182/commits/6ad6a4690249c5aaa30c32e284064f7b9e655b7e)